json/abilities.json

```diff
@@ -112,8 +154,8 @@
     "affects": "ABILITY: <span class=\"attribVal\">Unit Target</span><br />AFFECTS: <span class=\"attribVal\">Enemy Units</span><br />DAMAGE TYPE: <span class=\"attribVal\"><font color=\"#d61107\">Pure</font></span><br />PIERCES SPELL IMMUNITY: <span class=\"attribVal\">No</span><br />",
     "desc": "Feasts on the vital energies of an enemy unit, healing Bane and dealing damage.",
     "notes": "",
-    "dmg": "DAMAGE: <span class=\"attribVal\">90 / 160 / 230 / 300</span><br />",
-    "attrib": "HEAL AMOUNT: <span class=\"attribVal\">90 / 160 / 230 / 300</span>",
+    "dmg": "",
```

Comment:
I'm not sure if valve intended to remove this.  They seem to have broken these endpoints with the new update.